Tests/Core/ResolverTests.swift

```diff
@@ -131,8 +131,8 @@ class WrapTests: XCTestCase {
     }
 
     func testIsFulfilled() {
```

Comment:
Shouldn't the test name also change to `testIsSuccess()`?

Reply:
You’re right, thanks!